bndtools.core/resources/enroute/_plugin.xml

```diff
@@ -1,13 +1,5 @@
 <!-- ENROUTE TEMPLATES -->
 
-<extension point="bndtools.core.cnfTemplates">
-	<template id="enRoute" name="OSGi enRoute Configuration (alpha)"
-		doc="enroute/cnf.xml" paths="enroute/osgi.enroute.template/cnf"
-		priority="0" managed="ext/.*" ignores="/cache/" version="10" category="AAA/Core"
-		nobuild="true" icon="enroute/enroute-bw-16x16.png" />
-</extension>
-
-
```

Comment:
This seems to be the same as https://github.com/pkriens/bndtools/commit/c68e502fa2d13d6d744323229a276afe08a5e8ff